src/js/actions/zonePurgeCache.js

```diff
@@ -20,7 +20,26 @@ export function zonePurgeCacheError() {
     }
 }
 
-export function asyncZonePurgeCache(zoneId) {
+export function asyncZonePurgeCacheIndividualFiles(zoneId, files) {
+    return dispatch => {
+        dispatch(zonePurgeCache());
+
+        var formatedFiles = files.replace(/^\s+|\s+$/g,'').split(/\s+/);
+
+        v4ZonePurgeCache({zoneId: zoneId, files: formatedFiles}, function(response){
+            if(v4ResponseOk(response)) {
+                dispatch(zonePurgeCacheSuccess());
+                dispatch(notificationAddSuccess("container.purgeCacheCard.success", true));
```

Comment:
export function notificationAddSuccess(message, localized = false) {
    return notificationAdd("success", message, localized);
}

it gets localized messages. This is the notification bar at the bottom of the page which pops when something is successful. It's how it was done before also in cloudflare.com it's the same